common/changes/@uifabric/merge-styles/api-extractor-oufr_2018-10-18-21-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/merge-styles",
+      "comment": "Add api-extractor.json",
+      "type": "minor"
```

Comment:
>    "type": "minor" [](start = 2, length = 19)

Just curious, why is this a minor bump?  I don't see any changes in the **merge-styles** package

Reply:
The only changes in the merge-styles package were to package.json and config/api-extractor.json, to update the build task scripts to run api-extractor. This isn't a patch in the sense that it fixes a bug, so it's listed as a minor change.